src/components/profile/placements/Placements.tsx

```diff
@@ -1,64 +1,44 @@
 import React from "react";
-import Divider from "@material-ui/core/Divider";
-import ExpansionPanel from "@material-ui/core/ExpansionPanel";
-import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
-import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
-import Typography from "@material-ui/core/Typography";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { Details, Row, Col } from "nhsuk-react-components";
 import { PlacementPanel } from "./PlacementPanel";
 import { Placement } from "../../../models/Placement";
-import { makeStyles } from "@material-ui/core/styles";
+import styles from "./Placements.module.scss";
 
 interface IPlacementProps {
   placements: Placement[];
 }
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    width: "100%"
-  },
-  heading: {
-    fontSize: theme.typography.pxToRem(22),
-    fontWeight: theme.typography.fontWeightBold,
-    margin: "0px"
-  },
-  sectionPadding: {
-    padding: theme.typography.pxToRem(20)
-  }
-}));
-
 const Placements: React.FC<IPlacementProps> = ({ placements }) => {
-  const classes = useStyles();
+  const columnWidths: any[] = ["full", "full", "one-half", "one-third"];
+  let columnWidth = columnWidths[placements.length]
+    ? columnWidths[placements.length]
+    : "one-third";
+
   return (
     placements && (
-      <section className={classes.sectionPadding}>
-        <Divider />
-        <div>
-          <ExpansionPanel defaultExpanded={true}>
-            <ExpansionPanelSummary
-              expandIcon={<ExpandMoreIcon />}
-              aria-controls="panel1a-content"
-              id="panel1a-header"
-            >
-              <Typography className={classes.heading}>Placements</Typography>
-            </ExpansionPanelSummary>
-            <ExpansionPanelDetails>
-              <div className={classes.root}>
-                {placements.length === 0 ? (
-                  <div>You are not assigned to any placement</div>
-                ) : (
-                  placements.map(
-                    (
-                      placement: Placement,
-                      index: string | number | undefined
-                    ) => <PlacementPanel key={index} placement={placement} />
+      <>
```

Comment:
I see that we have only Details as fragment child.
Could you please check and remove the <> (fragment) id not required?